manifests/repo/rhel/erlang.pp

```diff
@@ -0,0 +1,23 @@
+# Makes sure that the Packagecloud repo for erlang is installed
+#
+# @api private
+class rabbitmq::repo::rhel::erlang (
+  $location          = "https://packagecloud.io/rabbitmq/erlang/el/${facts['os'][release][major]}/\$basearch",
```

Comment:
can you use Stdlib::HTTPSUrl here?
```suggestion
  Stdlib::HTTPSUrl $location = "https://packagecloud.io/rabbitmq/erlang/el/${facts['os'][release][major]}/\$basearch",
```

Reply:
This is used to determine baseurl in yum repos and it is not always https url but can be different format like http url or file:// or ftp:// . So we can't use HTTPSUrl and I'm not aware of any good alternatives.

Reply:
I'll close this conversation according to my previous comment.